docs/ml/creating-jobs.asciidoc

```diff
@@ -26,7 +26,18 @@ An _advanced job_ can contain multiple detectors and enables you to configure al
 job settings.
 
 {kib} can also recognize certain types of data and provide specialized wizards
-for that context. For example, if you use {filebeat-ref}/index.html[{filebeat}]
+for that context. For example, if you
+<<add-sample-data,added the sample web log data set>>, the following wizard
+appears:
+
+[role="screenshot"]
+image::ml/images/ml-data-recognizer-sample.jpg[A screenshot of the {kib} sample data web log job creation wizard]
+
+TIP: Alternatively, after you load a sample data set you can click
```

Comment:
Is it worth saying that this accessed from the 'Add data' section of the Kibana Home page? Otherwise it's not obviouus where this 'View data > ML jobs' button is.

Reply:
Good point, done!